scripts/runRBRTurskcdf2nc.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+import sys
+sys.path.append('C:\\projects\\python\\stglib')
```

Comment:
Please don't hard code paths for importing. The assumption is that the user will have installed these packages and are available to python. Same goes for other run scripts.